src/pages/app-development/index.md

```diff
@@ -1,6 +1,66 @@
 ---
-title: Adobe Commerce App Developer Guide Overview
-description: 
+title: Adobe Commerce App Developer's Guide Overview
+description: Learn how to make the transition from developing Commerce PHP extensions to developing Out-of-Process apps.
 ---
 
-# Adobe Commerce App Developer Guide Overview
+# Adobe Commerce App Developer's Guide Overview
+
+Developers have traditionally used PHP to create in-process extensions that add features, functionality, services, and integrations to Adobe Commerce. This model requires any new code to be compatible with upgrades, the server's PHP version, and many other essential server applications and services that Commerce uses.
+
+Adobe Developer App Builder uses out-of-process extensibility to avoid these compatibility issues. It provides a unified third-party extensibility framework for integrating and creating custom applications that extend Adobe Commerce. Since this extensibility framework is built on Adobe's infrastructure, developers can build custom microservices, and extend and integrate Adobe Commerce across other Adobe solutions and third-party integrations.
+
+## How do I port an extension into an app?
+
+Start by becoming familiar with the [App Builder documentation](https://developer.adobe.com/app-builder/docs/overview/) and [create an Adobe developer account](https://developer.adobe.com/app-builder/docs/overview/getting_access/).
+
+Once you've become comfortable with the Adobe I/O infrastructure, analyze  your current extensions and begin mapping its in-process features into the App Builder and Adobe I/O environment. Key areas to consider include:
+
+*  Frontend development
+*  APIs
+*  Plugins and observers
+*  Backend development
+*  Custom cron jobs
+*  Database data
+*  Filesystem
+
+### Frontend development
+
+[Spectrum](https://spectrum.adobe.com/page/principles/) provides all the tools you need to create the next generation of React-based applications. Adobe Commerce continues support of legacy [PWA Studio](https://developer.adobe.com/commerce/pwa-studio/) and [Luma](https://developer.adobe.com/commerce/frontend-core/) storefronts.
+
+### APIs
+
+[API Mesh for Adobe Developer App Builder](https://developer.adobe.com/graphql-mesh-gateway/) enables developers to integrate private and third-party APIs and other software interfaces with Adobe Commerce and other Adobe products using Adobe I/O.
```

Comment:
Let's add something else here:
Enables developers to integrate, proxy, adapt or extend 
I would add the motive: to easily fit the experience layer functionality according to any business requirements by wiring multiple apis together in a single endpoint whether that's Adobe API or 3rd party. - you can adapt this message how you see it

Also there's no private and third-party APIs as categorization in the same criteria
APIs are either public or private.
In terms of Adobe there's Aodbe apis (all public) and 3rd party as non adobe again all public